Content.Server/PowerCell/PowerCellSystem.cs

```diff
@@ -48,6 +49,14 @@ private void OnChargeChanged(EntityUid uid, PowerCellComponent component, Charge
         var frac = battery.CurrentCharge / battery.MaxCharge;
         var level = (byte) ContentHelpers.RoundToNearestLevels(frac, 1, PowerCellComponent.PowerCellVisualsLevels);
         appearance.SetData(PowerCellVisuals.ChargeLevel, level);
+
+        // If this power cell is inside a cell-slot, inform that entity that the power has changed (for updating visuals n such).
+        if (uid.TryGetContainer(out var container)
```

Comment:
Does ContainerSystem have a method for it?

Reply:
Yes it does, switched to using that.